src/api-service/dev-deploy.sh

```diff
@@ -25,12 +27,15 @@ rm -f __app__/onefuzztypes*.whl
 rm -rf local-pytypes
 cp -r ../pytypes local-pytypes
 pushd local-pytypes
-rm -f dist/*
+rm -rf dist build
```

Comment:
Double-checking, you really want to remove the `dist` directory? Asking since we only deleted its contents before.

Reply:
Yes, it's on purpose.  the glob missed '.' files